webapp/lib/playground.dart

```diff
@@ -0,0 +1,201 @@
+/**
+ * The entry point of the application.
+ */
+library coda.playground;
+
+import 'dart:html';
+
+import 'data_model.dart';
+import 'view_model.dart';
+
+Playground get playground => _playground;
+
+Playground _playground;
+
+void init() {
+  _playground = new Playground();
+}
+
+class Playground {
+  ButtonElement get saveButton => querySelector('#save-all-button');
+  TableElement get messageCodingTable => querySelector('#message-coding-table');
+
+  Dataset dataset;
+
+  Playground() {
+    // TODO: This is just for prototyping, the jsondataset will come from a server
+    loadDataset(new Dataset.fromJson(jsonDataset));
+  }
+
+  loadDataset(Dataset dataset) {
```

Comment:
This name seems to imply to me that it's only doing logical loader, whereas this code is also building a UI?

Reply:
Good point, thanks, renamed now.